src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export function Person({ person }) {}
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+
+export function Person({ person }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+
+      {person.age && (
+        <p className="Person__age">
+          `I am ${person.age}`
```

Comment:
```suggestion
          {`I am ${person.age}`}
```
you can use formatted strings only inside curvy brackets 😉